bin/ruby-build

```diff
@@ -1257,30 +1257,22 @@ build_package_ldflags_dirs() {
 }
 
 build_package_enable_yjit() {
-  local rustc_installed x86_arch rustc_ver
-
-  rustc_installed=""
-  if type -p rustc &>/dev/null; then
-    rustc_ver=`rustc --version`
-    # Some kind of built-in bash comparison for dotted version strings would be awesome.
-    if [[ "${rustc_ver}" == *"rustc 1."[6789]* ]]; then
-      rustc_installed="1"
-    fi
-  fi
+  # If YJIT is explicitly enabled or disabled, don't change the config
+  ([[ " ${RUBY_CONFIGURE_OPTS} " == *" --enable-yjit"* ]] || [[ " ${RUBY_CONFIGURE_OPTS} " == *" --disable-yjit"* ]]) && return 0;
```

Comment:
The `(...)` expression in bash spawns a subshell; let's avoid that since it's not necessary.
```suggestion
  if [[ " ${RUBY_CONFIGURE_OPTS} " == *" --enable-yjit"* || " ${RUBY_CONFIGURE_OPTS} " == *" --disable-yjit"* ]]; then
    return 0
  fi
```